<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::RubberBandDrag</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Image"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionImage_Window"/>
    <addaction name="actionOutput_Window"/>
    <addaction name="actionTemplate_Window"/>
   </widget>
   <widget class="QMenu" name="menuTheam">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionDark"/>
    <addaction name="actionDark_Orange"/>
    <addaction name="actionAdaptic"/>
    <addaction name="actionIntegrid"/>
    <addaction name="actionDefault"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTheam"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOutput_Window"/>
   <addaction name="actionTemplate_Window"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>213</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/logo</normaloff>:/icons/logo</iconset>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>202</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Template</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButton">
         <property name="text">
          <string>CommandLinkButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>259</width>
            <height>324</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Step 1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>48</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLCDNumber{
  background-color: rgb(0, 0, 0);
  border: 2px solid rgb(113, 113, 113);
  border-width: 2px;
  border-radius: 10px;
  color: rgb(0, 255, 0);
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="smallDecimalPoint">
                <bool>false</bool>
               </property>
               <property name="digitCount">
                <number>16</number>
               </property>
               <property name="segmentStyle">
                <enum>QLCDNumber::Flat</enum>
               </property>
               <property name="value" stdset="0">
                <double>12345678.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton_6">
                 <property name="text">
                  <string>Line</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_5">
                 <property name="text">
                  <string>Box</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>Ellipse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>298</width>
            <height>324</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Step 2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>RadioButton</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_3">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_2">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item>
              <widget class="QDial" name="dial"/>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_Image"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_Actual"/>
   <addaction name="actionZoom_Fit"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_Selection"/>
  </widget>
  <action name="actionImport_Image">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/image-import.png</normaloff>:/qss_icons/images/image-import.png</iconset>
   </property>
   <property name="text">
    <string>Import Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImage_Window">
   <property name="text">
    <string>Image Window</string>
   </property>
  </action>
  <action name="actionOutput_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/green</normaloff>:/images/green</iconset>
   </property>
   <property name="text">
    <string>Output Window</string>
   </property>
  </action>
  <action name="actionTemplate_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/red</normaloff>:/images/red</iconset>
   </property>
   <property name="text">
    <string>Template Window</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionDark_Orange">
   <property name="text">
    <string>Dark Orange</string>
   </property>
  </action>
  <action name="actionAdaptic">
   <property name="text">
    <string>Adaptic</string>
   </property>
  </action>
  <action name="actionIntegrid">
   <property name="text">
    <string>Integrid</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/Arrow-turn-left-icon.png</normaloff>:/qss_icons/images/Arrow-turn-left-icon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/Arrow-turn-right-icon.png</normaloff>:/qss_icons/images/Arrow-turn-right-icon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionZoom_Actual">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/zoom_actual_equal.png</normaloff>:/qss_icons/images/zoom_actual_equal.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Actual</string>
   </property>
   <property name="toolTip">
    <string>Zoom Actual</string>
   </property>
  </action>
  <action name="actionZoom_Fit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/zoom_fit.png</normaloff>:/qss_icons/images/zoom_fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom Fit</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/zoom_in.png</normaloff>:/qss_icons/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/zoom_out.png</normaloff>:/qss_icons/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_Selection">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/qss_icons/images/zoom_selection.png</normaloff>:/qss_icons/images/zoom_selection.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Selection</string>
   </property>
   <property name="toolTip">
    <string>Zoom Selection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOutput_Window</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>770</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionOutput_Window</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget_2</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionTemplate_Window</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTemplate_Window</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>732</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
